ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -48,7 +48,7 @@ class ClientQueryViewController: UITableViewController, Themeable {
 	var initialAppearance : Bool = true
 	private var observerContextValue = 1
 	private var observerContext : UnsafeMutableRawPointer
```

Comment:
@pablocarmu Is it necessary to configure key value observing using UnsafeMutableRawPointer? Can we avoid it? Check https://developer.apple.com/documentation/swift/cocoa_design_patterns/using_key-value_observing_in_swift

Reply:
@pablocarmu In commit 560917e21eafbf4511f77cb4ff0e8728835fcacd I only see that you have renamed reference to RefreshControl

Reply:
Seems that I forgot to include these changes in the commit, will do in the following one. Thanks!